.deployment/docker-compose-overrides.yml

```diff
@@ -5,8 +5,6 @@ services:
       - "DEEPFORGE_HOST=https://dev.deepforge.org"
       - "DEEPFORGE_INTERACTIVE_COMPUTE_HOST=https://dev-compute.deepforge.org"
     volumes:
-      - "$HOME/.deepforge/blob:/data/blob"
```

Comment:
Why did you remove this?

Reply:
Because it will be added by `yq` append from the main `docker-compose.yml`.